ts/components/conversation/media-gallery/AttachmentSection.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * @prettier
+ */
+import React from 'react';
+
+import { DocumentListItem } from './DocumentListItem';
+import { ItemClickEvent } from './events/ItemClickEvent';
+import { MediaGridItem } from './MediaGridItem';
+import { Message } from './propTypes/Message';
+import { missingCaseError } from '../../../util/missingCaseError';
+
+const styles = {
+  container: {
+    width: '100%',
+  },
+  header: {
+    fontSize: 14,
+    fontWeight: 'normal',
+    lineHeight: '28px',
+  } as React.CSSProperties,
+  itemContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+  } as React.CSSProperties,
+};
+
+interface Props {
+  i18n: (value: string) => string;
```

Comment:
Ah, I see what you were talking about regarding the simpler `i18n` declaration. Seems like this is something we should pull into a separate place.

Reply:
Agreed!